fare/config.py

```diff
@@ -141,7 +145,9 @@ def _(x):
 #: provided, the allowed hosts variable is set to localhost. In production it
 #: should be set to the correct host and it is strongly recommended to only
 #: route correct hosts to the application.
-APP_ALLOWED_HOSTS = ['fare.polito.it', 'localhost', '127.0.0.1']
+APP_ALLOWED_HOSTS = ['localhost', '127.0.0.1']
+if os.getenv('CURRENT_HOST') is not None:
```

Comment:
Why just here and not everywhere else? Define a proper way of handling the null cases